content/api/magnet-board.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Magnet Board API
```

Comment:
I don't think it really makes sense to call this a "board".

Reply:
Done :)